packages/IMAPClient-Core.package/ICEmail.class/instance/getDateFormatted.st

```diff
@@ -0,0 +1,21 @@
+accessing
+getDateFormatted
+
+	| date formatted |
+	
+	self date = '' ifTrue: [^''].
+	
+	date := DateAndTime fromSeconds: (
+			(MailMessage new timeFrom: self date) + (DateAndTime localOffset asSeconds) + ((Date newDay: 1 year: 1980) asSeconds) 
```

Comment:
I think it would be good to put an explaining comment here, because not everyone will immidiatly understand the code.